homebrew/chruby.rb

```diff
@@ -8,7 +8,8 @@ class Chruby < Formula
   head 'https://github.com/postmodern/chruby.git'
 
   def install
-    system 'make', 'install', "PREFIX=#{prefix}"
+    system 'make', 'install', 'install-zsh-completion', "PREFIX=#{prefix}"
```

Comment:
```suggestion
    zsh_completion.install 'integration/complation.zsh' => '_chruby'
```

Reply:
@ixti I think that would have to be done after `system 'make', 'install', "PREFIX=#{prefix}"` as an additional step.

I am not up-to-date with Homebrew's API, so I cannot comment on usage of `zsh_completion.install`.

Reply:
Oh. Indeed. I've missed that that line was calling `make install`. Good catch!

Reply:
@postmodern updated the suggestion :D